auraed/src/runtime/cell_service/executables/executable.rs

```diff
@@ -64,7 +64,8 @@ impl Executable {
             let mut stdout = BufReader::new(stdout).lines();
             while let Ok(Some(line)) = stdout.next_line().await {
                 let entered_span = span.take().expect("span").entered();
-                info!(level = "info", channel = log_channel.name, line);
+                //info!(level = "info", channel = log_channel.name, line);
```

Comment:
did you remove this on purpose? it's logging the spawned executable output.

Reply:
Its really not. It's "flooding" the entire output line by line with an enormous tracing line. To be honest it was obnoxious. Every line in the output would print a few hundred "meta" characters and then the content of the line. 

```
output 1
output 2
output 3
```

Was printed like:

```
runtime::cells::aurae::executable::output::something::meta::another {} log () line () date:: time -> output -> warn ::something::else -> output::runtime output 1
runtime::cells::aurae::executable::output::something::meta::another {} log () line () date:: time -> output -> warn ::something::else -> output::runtime output 2
runtime::cells::aurae::executable::output::something::meta::another {} log () line () date:: time -> output -> warn ::something::else -> output::runtime output 3
```

obviously I just made the log lines up, but you get the idea.